services/chat/api_send_messages.go

```diff
@@ -0,0 +1,159 @@
+package chat
+
+import (
+	"context"
+
+	"github.com/forPelevin/gomoji"
+
+	"github.com/status-im/status-go/eth-node/crypto"
+	"github.com/status-im/status-go/eth-node/types"
+	"github.com/status-im/status-go/protocol"
+	"github.com/status-im/status-go/protocol/common"
+	"github.com/status-im/status-go/protocol/protobuf"
+)
+
+type SendMessageResponse struct {
+	Chat     *Chat             `json:"chat"`
+	Messages []*common.Message `json:"messages"`
+}
+
+func (api *API) SendSticker(ctx context.Context, communityID types.HexBytes, chatID string, packID int32, hash string, responseTo string) (*SendMessageResponse, error) {
+	ensName, _ := api.s.accountsDB.GetPreferredUsername()
+
+	msg := &common.Message{
```

Comment:
Probably I would move the logic in messenger, so we keep the API fairly logic less, what do you think?
It would look something like:
```
SendSticker(....) (...) {
  response, err := ap.s.messenger.SendSticker(....)
  if err ...
  }
}

Reply:
I can do that change, I was just worried about increasing the number of functions in Messenger, but I guess I can just put all these in a new file.